src/System.Buffers.Primitives/System/Buffers/Buffer.cs

```diff
@@ -26,11 +26,48 @@ internal Buffer(OwnedBuffer<T> owner, int index, int length)
             _length = length;
         }
 
-        internal Buffer(T[] array, int index, int length)
+        public Buffer(T[] array)
```

Comment:
Why don't the short ctor just call the longet one? i.e. Buffer(T[] array) : this(array, 0, array.Length)?